src/main/webapp/js/search.js

```diff
@@ -5,7 +5,6 @@ $(function() {
       $searchButton = $("#searchButton"),
       contextPath = $("#contextPath").val(),
       loadImage;
-
```

Comment:
Please remove this change.

Reply:
I removed it.
Sorry for the unnecessary layers of modifications.